README.md

```diff
@@ -13,7 +13,7 @@ Add this to your `Cargo.toml`:
 
 ```toml
 [dependencies]
-threadpool = "1.0"
+threadpool = "1.2"
```

Comment:
This change isn't necessary; specifying `threadpool = 1.0` will still download the latest `1.x` version